Views/backend/connect/view/export/product/many_products_dialog.js

```diff
@@ -25,6 +25,11 @@ Ext.define('Shopware.apps.Connect.view.export.product.manyProductsDialog', {
     sourceIds: [],
 
     /**
+     * Contains the amount of products export all will export as batches
+     */
+    count: 0,
```

Comment:
why a global variable == global state? can you refactor it so that this is passed as argument